nix/frr-config/config/libexec/frr/docker-start

```diff
@@ -1,4 +1,9 @@
 #!/bin/sh
 
 source /libexec/frr/frrcommon.sh
-/libexec/frr/watchfrr $(daemon_list) --timeout 400
+
+/libexec/frr/watchfrr $(daemon_list) &
+
+/bin/frr-agent "${@}" &
```

Comment:
Running processes in the background means we would ignore them failing and not restart or fail the container. Could you please make a ticket to fix it? IMO it's quite critical to do sooner rather than later

Reply:
Yup.  On the todo list.  I'm going to replace this with supervisord